lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -209,27 +208,23 @@ class AutofillInfo {
   AutofillInfo({this.editingState, this.uniqueIdentifier, this.hint});
 
   /// The current text and selection state of a text field.
-  final EditingState editingState;
+  final EditingState? editingState;
 
   /// Unique value set by the developer.
   ///
   /// Used as id of the text field.
-  final String uniqueIdentifier;
+  final String? uniqueIdentifier;
 
   /// Attribute used for autofill.
   ///
   /// Used as a guidance to the browser as to the type of information expected
   /// in the field.
   /// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
-  final String hint;
+  final String? hint;
```

Comment:
Same as above comment: "If there is autofill info, it means this value is always sent. Therefore this should not be null."